frontend/src/scenes/insights/InsightTabs/FunnelTab/FunnelStepTable.tsx

```diff
@@ -0,0 +1,160 @@
+                    style={{ display: 'flex', alignItems: 'center' }}
+                >
+                    {isBreakdownChild ? (
+                        <div style={{ maxWidth: 270, wordBreak: 'break-word' }}>
+                            {formatBreakdownLabel(step.breakdown, cohorts)}
+                        </div>
+                    ) : (
+                        <div style={{ flexGrow: 1 }}>
+                            <InsightLabel
+                                seriesColor={color}
+                                fallbackName={step.name}
+                                hasMultipleSeries={steps.length > 1}
+                                breakdownValue={step.breakdown}
+                                hideBreakdown
+                                hideIcon
+                            />
+                        </div>
+                    )}
+                </SeriesToggleWrapper>
+            )
+        },
+        fixed: 'left',
+        width: 300,
+    })
+
+    columns.push({
+        title: 'Completed',
+        render: function RenderCompleted({}, step: FlattenedFunnelStep): JSX.Element {
+            return (
+                <ValueInspectorButton onClick={() => openPersonsModal(step, step.count + 1, step.breakdown_value)}>
+                    {step.count}
+                </ValueInspectorButton>
+            )
+        },
+        width: 30,
+    })
+
+    columns.push({
+        title: 'Conversion',
+        render: function RenderConversion({}, step: FlattenedFunnelStep): JSX.Element | null {
+            return step.order === 0 ? null : <span>{step.conversionRates.total}%</span>
+        },
+        width: 30,
+    })
+
+    columns.push({
+        title: 'Dropped off',
+        render: function RenderDropoff({}, step: FlattenedFunnelStep): JSX.Element | null {
+            return step.order === 0 ? null : (
+                <ValueInspectorButton
+                    onClick={() =>
+                        openPersonsModal(step, step.count + 1, step.breakdown_value)
+                    } /* TODO: does this modal support dropped off users? */
+                >
+                    {step.droppedOffFromPrevious}
+                </ValueInspectorButton>
+            )
+        },
+        width: 30,
+    })
+
+    columns.push({
+        title: 'From previous step',
+        render: function RenderDropoffFromPrevious({}, step: FlattenedFunnelStep): JSX.Element | null {
+            return step.order === 0 ? null : <span>{humanizeNumber(100 - step.conversionRates.fromPrevious, 2)}%</span>
+        },
+        width: 30,
+    })
+
+    columns.push({
+        title: 'Average time',
+        render: function RenderAverageTime({}, step: FlattenedFunnelStep): JSX.Element {
+            return <span>{humanFriendlyDuration(step.average_conversion_time, 2)}</span>
+        },
+        width: 30,
+    })
+
+    return (
+        <Table
```

Comment:
I think a quick fix for the overflow issue is to add `scroll={{x: tableScrollX}}` here with `    const { tableScrollX } = useIsTableScrolling('lg')` somewhere above